src/util/PathUtil.cpp

```diff
@@ -72,7 +73,14 @@ auto Util::hasXournalFileExt(const fs::path& path) -> bool {
     return extension == ".xoj" || extension == ".xopp";
 }
 
-auto Util::clearExtensions(fs::path& path, const std::string& ext) -> void {
+auto Util::hasPdfFileExt(const fs::path& path) -> bool {
+    return std::find(Util::PDF_EXTENSIONS.begin(), Util::PDF_EXTENSIONS.end(), path.extension()) !=
+           Util::PDF_EXTENSIONS.end();
+}
```

Comment:
Would it not be simpler to just add a `StringUtils::toLowerCase` as is done in the lambda `rm_ext` below? This would also cover all the cases like `.Pdf` an so on.

Reply:
Yeah, I thought about that before implementing this with a list of pdf extensions. At that point I thought it would be cleaner to do it like I did. I'm gonna change that.